src/components/NcModal/NcModal.vue

```diff
@@ -200,7 +200,7 @@ export default {
 			tabindex="-1">
 			<!-- Header -->
 			<transition name="fade-visibility" appear>
-				<div class="modal-header" data-theme-dark>
+				<div class="modal-header" v-bind="dataThemeDarkAttr">
```

Comment:
For me it would make the component better readable to move this not into a computed but keep it in the template:

```suggestion
				<div class="modal-header"
					:data-theme-light="brightBackdrop"
					:data-theme-dark="!brightBackdrop">
```